### PR TITLE
Gift Entry: Batches Tab "View More" link, Sorting

### LINKED ISSUE #5292: Gift Entry "Batches" Tab only shows 10 records, based on last added "Field to Display"
In the Gift Entry Object, when a user clicks on the Batches tab to see previous batches, they are shown 10 records only, and cannot see batches in that tab beyond that. When user chooses "Select Fields to Display" and saves, the List View is updated to show the last 10 records based on that field just added through the "Select Fields to Display" option, making an inconsistent and challenging user experience.

**Steps to Reproduce**
1. In Gift Entry, create more than 10 batches with at least 1 entry to allow them to save
2. Click the "Batches" tab and see 10 of the batches that you have created
3. Click the gear icon in the Batches window and select "Select Fields to Display"
4. Add a new field and save
5. See that the List View has been refreshed, and depending on the field, some batches may no longer be visible through this view

**Workaround**
To view all batches, go to NPSP Data Import Batches Object to see all batches and enter them.

### PR DESCRIPTION
**Pull request by bdvorachek**
# Critical Changes

# Changes
- We added a View More link to Gift Entry’s Batches tab. The link appears when Gift Entry has more than 10 batches. When you click View More, Gift Entry loads 10 more rows, and you can continue clicking the link until no further batches are available or the maximum of 2000 rows is reached.

# Issues Closed
- Closes #5292 
# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

File: src/lwc/geListView/geListView.js [hunk 1]
```diff
@@ -149,15 +151,17 @@ export default class geListView extends LightningElement {
     }
 
     get isShowStandardFooter() {
-        const showFooter =
-            this.showStandardFooter === 'true' ||
+        return this.showStandardFooter === 'true' ||
             this.showStandardFooter === true ||
```
Comment (vmartinov-sforg) [inline src/lwc/geListView/geListView.js:155]: Where does this.showStandardFooter get set? Also, why do we need to check against 3 values?

Comment (bdvorachek) [reply src/lwc/geListView/geListView.js:155]: I think we're compensating for the fact that @api values come across as Strings if they're set directly in the HTML for the containing component, but could be Booleans if they're bound to a variable.

On line 59 and 93 of geTemplates.html, I set the `show-standard-footer` attribute but don't give it a value, which [ends up defaulting it to Boolean true](https://developer.salesforce.com/docs/component-library/documentation/en/lwc/js_props_boolean). As long as we're consistent about doing Boolean properties like this, I agree with dropping the extra comparisons here.

Comment (vmartinov-sforg) [reply src/lwc/geListView/geListView.js]: Does this mean that the value will always be true?

Comment (bdvorachek) [reply src/lwc/geListView/geListView.js]: Yes. If we need to set it to `false`, we can omit the attribute entirely. And if we need to dynamically toggle it, we bind it to a variable and make sure that var is always a Boolean.

**Pull request merged**